### PR TITLE
expose skynet-token header

### PR DESCRIPTION
**Pull request by kwypchlo**
Skynet-Token is expected to be allowed and exposed for skynet kernel integration

### CODE REVIEW

**Review by DavidVorick — APPROVED** [review]

File: nginx/conf.d/include/cors-headers [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
-more_set_headers 'Access-Control-Expose-Headers: Content-Length,Content-Range,ETag,Accept-Ranges,Skynet-File-Metadata,Skynet-Skylink,Skynet-Proof,Skynet-Portal-Api,Skynet-Server-Api,upload-offset,upload-metadata,upload-length,tus-version,tus-resumable,tus-extension,tus-max-size,upload-concat,location';
+more_set_headers 'Access-Control-Allow-Headers: DNT,User-Agent,X-Requested-With,If-Modified-Since,If-None-Match,Cache-Control,Content-Type,Range,X-HTTP-Method-Override,upload-offset,upload-metadata,upload-length,tus-version,tus-resumable,tus-extension,tus-max-size,upload-concat,location,Skynet-Api-Key,Skynet-Token,Authorization';
+more_set_headers 'Access-Control-Expose-Headers: Content-Length,Content-Range,ETag,Accept-Ranges,Skynet-File-Metadata,Skynet-Skylink,Skynet-Proof,Skynet-Portal-Api,Skynet-Server-Api,Skynet-Token,upload-offset,upload-metadata,upload-length,tus-version,tus-resumable,tus-extension,tus-max-size,upload-concat,location';
```
Comment (ro-tex) [inline nginx/conf.d/include/cors-headers:5]: Do we need `Skynet-Token` twice?

Also, we might want to alphabetize these. There are so many of them now, it's getting very hard to navigate. F/U, though, no need to block this PR on that.

**Review by ro-tex — APPROVED** [review]

File: nginx/conf.d/include/cors-headers (hunk 1 above)
Comment (kwypchlo) [reply nginx/conf.d/include/cors-headers:5]: so one is in allow list, second one is in expose list, from what I understood we need both right?

Comment (ro-tex) [reply nginx/conf.d/include/cors-headers:5]: we don't need the browser to be able to send `skynet-token` to the server, just the other two.

Comment (ro-tex) [reply nginx/conf.d/include/cors-headers:5]: but it doesn't really matter, tbh

**Comment by kwypchlo** [comment]
not on a current roadmap

**Pull request closed without merging**